.changeset/small-ants-think.md

```diff
@@ -0,0 +1,5 @@
+---
+"@pantheon-systems/nextjs-kit": patch
+---
+
+DB-3782: Add Recipe Component for use in next-drupal-starter
```

Comment:
No need for the ticket number here.
Also try not to reference the starters in the packages because on `npm` they are not connected at all.